src/cls/rbd/cls_rbd.cc

```diff
@@ -108,6 +108,8 @@ cls_method_handle_t h_metadata_set;
 cls_method_handle_t h_metadata_remove;
 cls_method_handle_t h_metadata_list;
 cls_method_handle_t h_metadata_get;
+cls_method_handle_t h_get_snapshot_limit;
```

Comment:
Interesting: GitHub munged the part of your comment between the braces when translating to email. Therefore, your comment appeared in the email to be 'Minor: how about snapshot__limit?' I took the double underscore to be a simple typo.